code/game/area/areas.dm

```diff
@@ -521,7 +521,7 @@
 		for(var/atom/movable/AM in T.contents)
 			AM.change_area(old_area,src)
 
-var/list/ignored_keys = list("loc", "locs", "parent_type", "vars", "verbs", "type", "x", "y", "z","group","contents","air","light","areaMaster","underlays","lighting_overlay","corners")
+var/list/ignored_keys = list("loc", "locs", "parent_type", "vars", "verbs", "type", "x", "y", "z", "group", "contents", "air", "light", "areaMaster", "underlays", "lighting_overlay", "corners", "affecting_lights", "has_opaque_atom", "lighting_corners_initialised", "light_sources")
```

Comment:
Not really.

`resetVariables` is for pooling, which doesn't do anything with turfs because turfs can't be pooled.

This variable, `ignored_keys`, is used exactly *twice*, and both times it is to determine which variables *not* to copy over when "copying" turfs.

![](http://i.imgur.com/nYj5i4d.png)

Note that the image is slightly out of date.